src/main/resources/theme/play/teammatchmaking/matchmaking_member_card.fxml

```diff
@@ -3,97 +3,45 @@
+        <HBox>
+            <HBox alignment="CENTER_LEFT" spacing="4.0" HBox.hgrow="ALWAYS" >
+                <children>
+                    <Label fx:id="clanLabel" styleClass="clan" text="[CLAN]:" />
+                    <Label fx:id="usernameLabel" styleClass="member-name" text="&lt;Username&gt;" />
+                    <ImageView fx:id="avatarImageView" fitHeight="20.0" fitWidth="40.0" preserveRatio="true">
+                        <HBox.margin>
+                            <Insets left="5.0"/>
+                        </HBox.margin>
+                    </ImageView>
+                </children>
+                <padding>
+                    <Insets top="2.0" />
+                </padding>
             </HBox>
-
-            <padding>
-                <Insets left="10"/>
-            </padding>
-        </VBox>
-
-        <!-- FACTION BUTTONS -->
-        <HBox alignment="CENTER" spacing="0.0">
+            <JFXButton fx:id="kickPlayerButton" onAction="#onKickPlayerButtonClicked" text=""
+                       styleClass="icon-button" />
+        </HBox>
+        <HBox spacing="5.0" alignment="CENTER_LEFT">
             <children>
-                <ToggleButton fx:id="aeonButton" mnemonicParsing="false" onAction="#onFactionButtonClicked"
-                              selected="true" text="">
-                    <styleClass>
-                        <String fx:value="icon-button"/>
-                        <String fx:value="aeon"/>
-                    </styleClass>
-                </ToggleButton>
-                <ToggleButton fx:id="cybranButton" mnemonicParsing="false" onAction="#onFactionButtonClicked"
-                              selected="true" text="">
-                    <styleClass>
-                        <String fx:value="icon-button"/>
-                        <String fx:value="cybran"/>
-                    </styleClass>
-                </ToggleButton>
-                <ToggleButton fx:id="uefButton" mnemonicParsing="false" onAction="#onFactionButtonClicked"
-                              selected="true" text="">
-                    <styleClass>
-                        <String fx:value="icon-button"/>
-                        <String fx:value="uef"/>
-                    </styleClass>
-                </ToggleButton>
-                <ToggleButton fx:id="seraphimButton" mnemonicParsing="false" onAction="#onFactionButtonClicked"
-                              selected="true" text="">
-                    <styleClass>
-                        <String fx:value="icon-button"/>
-                        <String fx:value="seraphim"/>
-                    </styleClass>
-                </ToggleButton>
-
-                <Label fx:id="refreshingLabel" styleClass="icon" text="" visible="false"/>
+                <Label fx:id="ratingLabel" text="%teammatchmaking.rating" styleClass="player-card-text" />
+                <Label fx:id="gameCountLabel" text="%teammatchmaking.gameCount" styleClass="player-card-text" />
```

Comment:
rating and game count was removed on purpose

Reply:
see my comment above